lib/dotcom_web/components/transit_icons.ex

```diff
@@ -0,0 +1,50 @@
+defmodule DotcomWeb.Components.TransitIcons do
```

Comment:
📓 I put these in here for now. They aren't really customizable. We should figure out how we want to do that on a broader scale later. E.g., we could offer three different sizes.